airflow/providers/microsoft/azure/operators/container_instances.py

```diff
@@ -19,6 +19,7 @@
 import re
 from collections import namedtuple
 from time import sleep
+from datetime import timedelta
```

Comment:
```suggestion
from datetime import timedelta
from time import sleep
```

to fix static checks